tasks/22_remove-falsy-values/remove-falsy-values.js

```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line max-len
+export const removeFalsyValues = ( arr ) => arr.filter( (item) => !!item === true );
```

Comment:
you don't need to compere as by returning `!!item` filter function will know that need to skip element. Also in this case you can remove `// eslint-disable-next-line max-len`

Reply:
Refactored.